controllers/get-articles-by-id.controller.js

```diff
@@ -4,9 +4,13 @@ exports.getArticlesById = (req, res, next) => {
   const id = req.params.article_id;
   fetchArticlesById(id)
     .then((articles) => {
-      res.status(200).send({ articles});
+      res.status(200).send({ articles });
     })
     .catch((err) => {
+      if (err.status === 404) {
```

Comment:
This should be in either your model or your error handling and not your controller.